docs/developer/base/logs-crawlers.md

```diff
@@ -0,0 +1,47 @@
+# Log Crawlers
+
+## Overview
+
+Some systems expose their logs from HTTP endpoints instead of files that the Logs Agent can tail.
+In such cases you can create an agent integration to crawl the endpoints and submit the logs.
+
+This diagram shows how a logs crawler agent integration that fits into the agent.
```

Comment:
```suggestion
This diagram shows how a logs crawler agent integration fits into the agent.
```

maybe: "This diagram shows how a logs crawler agent integration fits into the agent's log submission pathway."

Or something along those lines. I'll defer to docs to see what they think is clear.

Reply:
I'm also not happy with how I phrased it, waiting for docs to weigh in :)

Reply:
Closing this, see [thread with docs](https://github.com/DataDog/integrations-core/pull/18332/files#r1727253961).